src/EditorFeatures/CSharpTest/CodeActions/GenerateFromMembers/GenerateConstructor/GenerateConstructorTests.cs

```diff
@@ -93,7 +93,7 @@ protected override object CreateCodeRefactoringProvider(Workspace workspace)
             await TestAsync(
 @"class Z { [|public int A { get ; private set ; } public string B { get ; private set ; }|] } ",
 @"class Z { public Z ( int a , string b ) { this . A = a ; this . B = b ; } public int A { get ; private set ; } public string B { get ; private set ; } } ",
-index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyMemberAccessWithThisOrMe, "C#"), true } });
+index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyPropertyAccess, LanguageNames.CSharp), true } });
```

Comment:
I added a helper for the newly added test cases, this was just updating an already present test, but I'll expand the helper to be more helpful.